packages/core/src/Image.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components'
 
 const Image = styled.img.attrs(props => ({
-  height: 'auto'
+  height: props.height || 'auto'
```

Comment:
This had to be affecting more than a few use cases, right?

Reply:
it broke Placeholder image with styled-components v4